setup.py

```diff
@@ -5,14 +5,16 @@
     include_package_data=True,
     long_description_content_type="text/markdown",
     install_requires=[
-        "dbus-python==1.2.16",
-        "Flask==1.1.2",
-        "Flask-SocketIO==5.0.1",
-        "eventlet==0.31.0",
+        "Flask==2.1.3",
+        "Flask-SocketIO==4.3.1",
+        "eventlet==0.33.3",
         "blessed==1.17.10",
         "pynput==1.7.1",
         "psutil==5.6.6",
         "cryptography==3.3.2",
+        "jinja2==3.0.3",
+        "itsdangerous==2.0.1",
+        "Werkzeug==2.0.3",
```

Comment:
Out of curiosity, does pinning a newer version of Flask not pull along these dependencies?

Reply:
I think Flask pulls its SocketIO addition with it, but again, I handpicked that one because it was the only one that would properly start up the Webapp without errors.

Reply:
Alright, thanks for the explanation. I'm fine with pinning these for now. I may revisit these dependencies once I look into updating Flask further since Flask should, ideally, manage these as tertiary dependencies for us.